localisation/00_zuwc_united_localisation.csv

```diff
@@ -977,14 +977,14 @@ turn_yourself_into_monstrous_undead;Turn yourself into [Root.GetMonstrousUndeadN
 EVTOPTA_WCDAC_5000;I have to find out what's going on.;;Ich muss herausfinden, was da vor sich geht.;;;;;;;;;;;x
 EVTOPTB_WCDAC_5000;Suppress them!;;Unterdr�ckt sie!;;;;;;;;;;;x
-EVTDESC_WCDAC_5005;As soon as the first stars began to light up in the sky of Azeroth, you collected all the pale-skinned of your clan at the big fire.\n\nYou shouted at them absolutely any threats, but they only looked indifferently at you with dark-clouded eyes. "The Hour of Twilight is near. We hear their voice and you, [Root.GetFirstName], do not change anything anymore", they almost chorus.\n\nFeeling your helplessness, you realised that with them it seems all is over. Nothing prevents you from giving your leader an order to exterminate them.;;Kaum, dass die ersten Sterne den Himmel Azeroths zu leuchten begannen, versammelt Ihr alle hellh�utigen Mitglieder Eures Clans am gro�en Lagerfeuer.\n\nIhr bedroht sie, schreit sie an, aber sie sehen Euch nur mit dunklen, stumpfen Augen an. �Die Stunde des Zwielichts ist nahe. Wir h�ren Ihre Stimme und Ihr, [Root.GetFirstName], k�nnt nichts daran �ndern� erklingen sie fast gleichzeitig.\n\nMit Schrecken wird Euch Eure Machtlosigkeit klar. Nichts hindert Euch mehr daran, Eurem Marschall den Befehl zu geben, sie hinzurichten.;;;;;;;;;;;x
+EVTDESC_WCDAC_5005;As soon as the first stars began to light up in the sky of Azeroth, you collected all the pale-skinned of your clan at the firepit.\n\nYou shouted countless threats at them, but they only looked indifferently at you with dark-clouded eyes. "The Hour of Twilight is near. We hear their voice and you, [Root.GetFirstName], do not change anything anymore", they almost chorus.\n\nFeeling your helplessness, you realised that to them it seems everything is over. Nothing prevents you from giving your leader an order to exterminate them.;;Kaum, dass die ersten Sterne den Himmel Azeroths zu leuchten begannen, versammelt Ihr alle hellh�utigen Mitglieder Eures Clans am gro�en Lagerfeuer.\n\nIhr bedroht sie, schreit sie an, aber sie sehen Euch nur mit dunklen, stumpfen Augen an. �Die Stunde des Zwielichts ist nahe. Wir h�ren Ihre Stimme und Ihr, [Root.GetFirstName], k�nnt nichts daran �ndern� erklingen sie fast gleichzeitig.\n\nMit Schrecken wird Euch Eure Machtlosigkeit klar. Nichts hindert Euch mehr daran, Eurem Marschall den Befehl zu geben, sie hinzurichten.;;;;;;;;;;;x
```

Comment:
I'd replace leader with marshal/respective job title, since it would be most likely be his job to do this kind of thing, wouldn't it?
> Nothing prevents you from giving your leader an order to exterminate them.

And I'd replace "an order" with "the order".